lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -50,6 +51,12 @@ def do_execute(machine, command, options={})
           end
         }
 
+        if options[:metainfo]
```

Comment:
Do you want meta info about your command failure or not -- on second thought this is a bad name. How about `options[:verbose_error]` ?